infra/cifuzz.py

```diff
@@ -40,49 +51,114 @@ def main():
   build_fuzzer_parser = subparsers.add_parser(
       'build_fuzzers', help='Build an OSS-Fuzz projects fuzzers.')
   build_fuzzer_parser.add_argument('project_name')
-  build_fuzzer_parser.add_argument('repo_name')
+  build_fuzzer_parser.add_argument('github_repo_name')
   build_fuzzer_parser.add_argument('commit_sha')
 
   run_fuzzer_parser = subparsers.add_parser(
       'run_fuzzers', help='Run an OSS-Fuzz projects fuzzers.')
   run_fuzzer_parser.add_argument('project_name')
+  run_fuzzer_parser.add_argument('fuzz_time')
   args = parser.parse_args()
 
+  # Get the shared volume directory and creates required directory.
+  if os.environ['GITHUB_WORKSPACE']:
+    git_workspace = os.path.join(os.environ['GITHUB_WORKSPACE'], 'storage')
+    if not os.path.exists(git_workspace):
+      os.mkdir(git_workspace)
+    out_dir = os.path.join(os.environ['GITHUB_WORKSPACE'], 'out')
+    if not os.path.exists(out_dir):
+      os.mkdir(out_dir)
+  else:
+    return Status.ERROR.value
+
   # Change to oss-fuzz main directory so helper.py runs correctly.
   if os.getcwd() != helper.OSSFUZZ_DIR:
     os.chdir(helper.OSSFUZZ_DIR)
 
   if args.command == 'build_fuzzers':
-    return build_fuzzers(args) == 0
-  if args.command == 'run_fuzzer':
-    print('Not implemented')
-    return False
-  print('Invalid argument option, use build_fuzzers or run_fuzzer.')
-  return False
+    return build_fuzzers(args, git_workspace, out_dir).value
+  if args.command == 'run_fuzzers':
+    return run_fuzzers(args, out_dir).value
+  return Status.SUCCESS.value
 
 
-def build_fuzzers(args):
+def build_fuzzers(args, git_workspace, out_dir):
   """Builds all of the fuzzers for a specific OSS-Fuzz project.
 
+  Args:
+    args: List of args passed in to cifuzz.py build_fuzzers parser.
+    git_workspace: The location in the shared volume to store git repos.
+    out_dir: The location in the shared volume to store output artifacts.
+
+  Returns:
+    A Status enum representing the state of the build.
+  """
+  # TODO: Modify build_specified_commit function to return src dir.
+  src = '/src'
+
+  inferred_url, oss_fuzz_repo_name = build_specified_commit.detect_main_repo(
+      args.project_name, repo_name=args.github_repo_name)
+  if not inferred_url or not oss_fuzz_repo_name:
+    print('Error: Repo URL or name could not be determined.', file=sys.stderr)
```

Comment:
Just use python's logger module instead of printing to stderr

Reply:
Done.